requirements.txt

```diff
@@ -23,6 +23,7 @@ q2-types
 q2cli
 q2templates
 qiime2
+scikit-bio<=0.5.6
```

Comment:
Given your comment in the PR: shouldn't Q2 and the linked Q2 packages be fixed here to 2021.11? Or does fixing scikit-bio do that indirectly?

Reply:
I think some other dependency in this list already causes it to be 2021.11 so no need for us to pin. However, `scikit-bio` is not pinned anywhere in the qiime2's own dependencies so it would always resolve to the newest - that's why I had to pin it here (it's not related to Q2 being 2021.11 and it's not the reason for Q2 being 2021.11).